Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py

```diff
@@ -287,7 +439,73 @@ def get_search_result_path(search_response: List[Any]) -> str:
     return next(x['location'] for x in search_response if x['dataType'] == 'rows')
 
 
-def get_alerts(
+def create_output(columns: List[str], rows: List[List[Any]]) -> List[str]:
+    """Maps Varonis response to xsoar
+
+    :type columns: ``List[str]``
+    :param columns: Xsoar columns
+
+    :type rows: ``List[List[Any]]``
+    :param rows: Output values retrieved from Varonis
+
+    :return: The list of xsoar objects
+    :rtype: ``List[str]``
+    """
+    outputs: List[Any] = []
+    for row in rows:
+        output = dict()
+        for i in range(0, len(columns)):
+            output[columns[i]] = row[i]
+        outputs.append(output)
+    return outputs
+
+
+def try_convert(item, converter, error=None):
```

Comment:
Pycharm is complaining that error is missing in docstring

Reply:
fixed, thanks